lib/java-extras/src/main/java/org/triplea/performance/PerfTimer.java

```diff
@@ -15,7 +15,7 @@
  * <p>Example usage with auto-close try block:
  *
  * <pre>{@code
- * try(PerfTimer timer = PerfTimer.startTimer("timer_name_0")) {
+ * try(final PerfTimer timer = PerfTimer.time("timer_name_0")) {
```

Comment:
Is this change intended?

Reply:
Yes, the previous comment was wrong. Just fixing the comment along the way.

Reply:
The final keyword was left out on purpose. It doesn't do anything at all.
```java
try (PerfTimer timer = PerfTimer.time("timer_name_0")) {
  timer = PerfTimer.time("another timer"); // wont compile, timer is final
}
```

Reply:
Sorry, the main purpose of this change is to use `time()` instead of `startTimer()` since the later doesn't exist. Removed the `final` now.

Reply:
No worries, just a small detail I didn't know myself for a long time :D